libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -415,4 +425,61 @@ void AP_Camera_Backend::log_picture()
 }
 #endif
 
+bool AP_Camera_Backend::set_tracking_external(TrackingType tracking_type, const Vector2f& p1, const Vector2f& p2) {
+#if AP_CAMERA_TRACKING_ENABLED
+    if (tracker == nullptr) {
+        return false;
+    }
+    return tracker->set_tracking(tracking_type, p1, p2, _params.track_sysid, _params.track_compid, _cam_info);
+#else
+    return false;
+#endif
+}
+
+bool AP_Camera_Backend::set_tracking(TrackingType tracking_type, const Vector2f& p1, const Vector2f& p2) {
+    if (_params.track_enable == 1) {
+        return set_tracking_external(tracking_type, p1, p2);
+    } else {
+        return set_tracking_internal(tracking_type, p1, p2);
+    }
+}
+
+void AP_Camera_Backend::handle_message_camera_information(mavlink_channel_t chan, const mavlink_message_t &msg)  {
```

Comment:
I think we will face conflicts when using a mavlink enabled camera (e.g. CAMx_TYPE = MavlinkCamV2) because we will also receive a camera_information message from these cameras.